db/migrate/20100929044155_add_persistence_token_to_users_and_admins.rb

```diff
@@ -1,13 +0,0 @@
-class AddPersistenceTokenToUsersAndAdmins < ActiveRecord::Migration
```

Comment:
Mixing in apples and oranges is generally not the way to simplify things, at least for reviewers, so don't do this again! *waggles finger* :) 
On this occasion, it wasn't difficult to see what was going on, so I didn't remove the "Ready to merge" label and just wanted to make sure this wasn't a mistake.